moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -6,38 +6,67 @@ add_library(${MOVEIT_LIB_NAME}
   src/bullet_integration/bullet_cast_bvh_manager.cpp
   src/collision_env_bullet.cpp
   src/bullet_integration/bullet_bvh_manager.cpp
+  src/bullet_integration/contact_checker_common.cpp
+  src/bullet_integration/ros_bullet_utils.cpp
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} moveit_collision_detection
-    ${catkin_LIBRARIES} ${urdfdom_LIBRARIES}
-    ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES}
-    ${BULLET_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  rclcpp
+  rmw_implementation
+  urdf
+  urdfdom
+  urdfdom_headers
+  Boost
+  visualization_msgs
+)
+target_link_libraries(${MOVEIT_LIB_NAME}
+  moveit_collision_detection  
+  ${BULLET_LIBRARIES}
+)
 
 add_library(collision_detector_bullet_plugin src/collision_detector_bullet_plugin_loader.cpp)
 set_target_properties(collision_detector_bullet_plugin PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(collision_detector_bullet_plugin ${catkin_LIBRARIES} ${MOVEIT_LIB_NAME})
+ament_target_dependencies(collision_detector_bullet_plugin
+  rclcpp
+  urdf
+  visualization_msgs
+  pluginlib
+  rmw_implementation
+)
+target_link_libraries(collision_detector_bullet_plugin
+  ${MOVEIT_LIB_NAME}
+  moveit_planning_scene
+)
 
 install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_bullet_plugin
```

Comment:
The plugin needs to be exported here for symlink support. See https://github.com/ros-planning/moveit2/pull/339/files#diff-049302fd4031b4efb956b186b90e32e634cc629a6a7740c3feb205b805881522